buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -168,6 +168,33 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "Create Helix Test Jobs",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "c6c4c611-aa2e-4a33-b606-5eaba2196824",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "solution": "src\\tests.builds",
```

Comment:
Rather than using an "MSBuild" step, can you use a "cmd" step like you did for Linux / OSX.  I'd rather reduce the way that we do things and not introduce the possibility that we are using two different msbuild versions here on Windows vs other.